receiver/prometheusreceiver/factory.go

```diff
@@ -40,6 +40,7 @@ func NewFactory() receiver.Factory {
 	// This includes ScrapeManager lib that is used by the Promethes receiver.
 	// We need to set the validation scheme to _something_ to avoid panics, and
 	// UTF8 is the default in Prometheus.
+	//nolint:staticcheck // TODO: Figure out how to remove the deprecated global.
```

Comment:
`UTF8Validation` is the default now: https://github.com/prometheus/common/blob/95acce133ca2c07a966a71d475fb936fc282db18/model/metric.go#L49.  You should be able to just remove this?

Reply:
Should I wait for this to be done before merging @ArthurSens ?